### DISCUSSION TITLE
Setting text color

### DISCUSSION THREAD

**Discussion by troncoso** [ueberdosis/tiptap#1532] [General Questions]
I can't find anywhere in the documentation or existing issues anything about setting font color. There is a single existing unanswered question about this in the discussions. Is this already available in an existing tiptap or community plugin? Or is implementing it so simple, no one has ever bothered discussing it? I mean, adding the color to the node is probably pretty easy, but the more complex issue is a proper color picker

**Comment by hanspagel** [comment]
We provide a TextStyle extension, which adds support for span-tags. Those tags can have different style attributes. An example is the FontFamily extension. Try that one, copy the source code and replace font-family with color.

It’s not obvious, but very poweful to work that way.
